src/Pulsar.Client/Internal/PulsarHttpClient.fs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Pulsar.Client.Internal
+
+open System.Net.Http
+open Pulsar.Client.Api
+open System
+
+//  This class is mainly aimed for http lookup service
```

Comment:
Can you please add a comment, that in Java it's just HttpClient

Reply:
Fixed